float32 map_origin_pos_x
float32 map_origin_pos_y
geometry_msgs/Point goal

